### PR TITLE
Inventories move cgroup to readme

### PR DESCRIPTION
**Pull request by eroussy**
Move the description of the cgroups in the README instead of directly in the inventories

### CODE REVIEW

File: inventories/seapath_cluster_definition_example.yml [hunk 1]
```diff
@@ -136,16 +136,9 @@ all:
                         #OPTIONAL
                         logstash_server_ip: 10.10.10.10
 
-                        # Realtime/CPUs cgroups isolation configuration
-                        # All those parameters are optional.
-                        isolcpus: "2-5,14-17" # CPUs to isolate
-
-                        cpusystem: "0,12" # CPUs reserves for system
```
Comment (dupremathieu) [inline inventories/seapath_cluster_definition_example.yml:143]: Why not keep this variable in the example?
To me, the idea of these inventories is to give a usage example of all the possible variables. Do you want to change this?

Comment (eroussy) [reply inventories/seapath_cluster_definition_example.yml:143]: I agree that these inventories must provide usage examples.
However, they are also a starting point for people that want to implement SEAPATH : They copy this inventory and modify it line by line.
I think it is a good idea to keep this inventory simple for those people.

Particularly, these cpu variables are advanced variables and should not be proposed to beginners.
I think the best would be to have different kind of inventories (basic, networking, cgroups, etc ...) for examples, but I will not do that now.

Comment (dupremathieu) [reply inventories/seapath_cluster_definition_example.yml:143]: We have the `seapath_demo_cluster_definition_example.yml` inventory for beginners.

File: inventories/README.md [hunk 2]
```diff
@@ -16,25 +16,13 @@ It is possible with Ansible to provides [multiple inventories files](https://doc
 * the physical machines
 * the cluster internal network (Open vSwitch topology)
 * the virtual machines
-## SEAPATH Ansible inventories examples description
```
Comment (dupremathieu) [inline inventories/README.md:19]: I preferred keeping SEAPATH Ansible inventories examples description section before Advanced variables.

**Pull request closed without merging**